Casks/f/flox.rb

```diff
@@ -0,0 +1,54 @@
+cask "flox" do
+  arch arm: "aarch64", intel: "x86_64"
+
+  version "1.0.3"
+  sha256 arm:   "08c5c45ba72809c208405002f0200246fc90fc44cf09ac7298bc2c3258ef3b4f",
+         intel: "e03da4432d564fa9099ac256f472885eb8b6ab074b6d6a70378396b2b4641464"
+
+  url "https://downloads.flox.dev/by-env/stable/osx/flox-#{version}.#{arch}-darwin.pkg"
+  name "flox"
+  desc "Manages environments across the software lifecycle"
+  homepage "https://flox.dev/"
+
+  livecheck do
+    url "https://downloads.flox.dev/by-env/stable/LATEST_VERSION"
+    strategy :page_match do |page|
+      match = page.match(/(\d+(?:\.\d+)+)/)
+      match ? match[0] : nil
+    end
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$\.zip/i)
```

Update to align with standard pattern for `livecheck` blocks.

Reply:
I don't think that regex would match? There's no `.zip` anywhere. The file we're pulling for is literally a bare version string and that's it. In this example it says "1.0.3" that's the whole file.

Reply:
FWIW, I assumed would be really to just read a string, but it was remarkably obtuse in trying to figure out how to get it to work. If there's a shorter/more straightforward way of doing it -- happy to change.

Reply:
Sorry yes that was a mistake on my part. I will update the suggestion